cmd/crank/install.go

```diff
@@ -265,7 +265,7 @@ func (c *installProviderCmd) Run(k *kong.Context, logger logging.Logger) error {
 }
 
 func warnIfNotFound(err error) error {
-	serr, ok := err.(*apierrors.StatusError)
+	serr, ok := err.(*apierrors.StatusError) //nolint:errorlint // we need to be able to extract the underlying typed error
```

Comment:
Not sure why we need the `nolint` here. Could you elaborate?

Reply:
It was a lint error asking to use `errors.As` but that only returns a bool and we'd still have to cast it to `*apierrors.StatusError` 
```
cmd/crank/install.go:268:14: type assertion on error will fail on wrapped errors. Use errors.As to check for specific errors (errorlint)
	serr, ok := err.(*apierrors.StatusError)
```